config/linux-nitropad-x.config

```diff
@@ -1989,6 +2010,7 @@ CONFIG_FB_DEFERRED_IO=y
 # CONFIG_FB_IMSTT is not set
 # CONFIG_FB_VGA16 is not set
 CONFIG_FB_VESA=y
+# CONFIG_FB_EFI is not set
```

Comment:
Something is wrong here.

Reply:
fixed under [3a648df](https://github.com/linuxboot/heads/pull/1522/commits/3a648df8ce20b68dca1348a170cb2b7281b6bc5d)